azure-pipelines.yml

```diff
@@ -293,6 +293,39 @@ jobs:
       channels: # for Roslyn jobs we want to check .NET Core 3.1 and 5.0 only
         - main
 
+  # Ubuntu 1804 x64 ILLink Benchmarks
+  - template: /eng/performance/benchmark_jobs.yml
+    parameters:
+      osName: ubuntu
+      osVersion: 1804
+      kind: illink
+      architecture: x64
+      pool:
+        vmImage: ubuntu-latest
+      machinePool: Tiger
+      queue: Ubuntu.1804.Amd64.Tiger.Perf # using a dedicated private Helix queue (perftigers)
```

Comment:
This queue and the one for the Windows job need to match the other queue names in the public correctness section. We cannot use the queues from the private jobs area, as those queues are only available for internal runs.